tc/sgx/trusted_worker_manager/enclave/enclave_data.h

```diff
@@ -65,13 +66,19 @@ class EnclaveData {
     // allocated storage for the sealed data. There are a number of ways
     // to do this; for now, the constant will be good enough.
     static const size_t cMaxSealedDataSize = 8192;
-    static const size_t cMaxPublicDataSize = 4096;
+    //static const size_t cMaxPublicDataSize = 4096;
+
+    // size of RSA key signature is 2048
+    static const size_t cMaxPublicDataSize = 6144;
```

Comment:
How did we arrive at 6144 ?

Reply:
If you need a larger buffer because of base64, the formula is
(bytes * 4 / 3) + 2 +1
The +2 is because of the 0 to 2 "=" padding bytes. This can be computed exactly if needed.
The +1 is if there is a NUL terminator byte
So I come up with 2048*4/3 +2 + 1 = 2733

Reply:
I added 2048 bytes(to account encryptionKeySignature) to  cMaxPublicDataSize(before 4096 bytes) which sum up to 6144. 
the base64 encoded version of the signature was less than 2048, hence i chose to add 2048 to the cMaxPublicDataSize buffer size.
@Dan yes we can allot max buffer size equals to 2733 bytes as you suggested.